docs/coding-standards.md

```diff
@@ -4,48 +4,40 @@ title: Coding Standards
 permalink: /coding-standards/
 ---
 
-Write accurate [PHPDoc](http://en.wikipedia.org/wiki/PHPDoc) styled code comments.
+## Validating with PHP Code Sniffer
 
-Write real Classes or Namespaced functions.
+Instead of reading any of this why not just let the computer nag you? From the Pressbooks plugin directory:
 
-[PHP Namespaces](https://secure.php.net/manual/en/language.namespaces.php) have been available since 2009. Namespaces are not a new concept. We use them.
+1. `composer install --dev`
+2. `composer standards`
 
-Our namespace is: `\Pressbooks\`
+Bonus: You can sometimes automatically fix errors by running `vendor/bin/phpcbf --standard=phpcs.ruleset.xml /path/to/your/file`
 
- * If your Class isn't an Object like `\WP_User`, `\WP_Dependencies`, `\WP_Query` etc., write a library of functions.
- * If your Class is a bunch of Static methods and nothing else, write a library of functions.
- * Afraid of function name collisions? See [Namespaces](https://secure.php.net/manual/en/language.namespaces.php).
+## Pressbooks Coding Standards (Mandatory)
+
+We enforce [Human Made Coding Standards](https://github.com/humanmade/coding-standards) with the following small tweaks.
 
-Functions are `written_like_this()`.
-Class methods are `writtenLikeThis()`.
+ + Use camelCase for class methods & properties, uppercase for class constants, snake_case everywhere else.
+ + [PHP Sessions](http://php.net/manual/en/book.session.php) are allowed.
+ 
+### Write Classes or Namespaced functions, stay out of global space!
 
-Rules of thumb:
+[PHP Namespaces](https://secure.php.net/manual/en/language.namespaces.php) have been available since 2009. Namespaces are not a new concept. We use them.
 
- * `getFooBar()` is when you are getting something associated with an object -- it implies the set is already defined.
- * `loadFooBar()` is when you are loading from an external source, like a file or a database.
- * `isFooBar()` or `hasFooBar()` is when a method returns a boolean value.
- * `createFooBar()` is when a method creates new files or assets.
+Our namespace is: `\Pressbooks\`
 
-Don't write `public function foo()`, just write `function foo()`. Public is implied.
-Order of methods in a class:
+ * If your Class isn't an Object like `\WP_User`, `\WP_Dependencies`, `\WP_Query` etc., write a library of functions.
+ * If your Class is a bunch of Static methods and nothing else, write a library of functions.
+ * Afraid of function name collisions? See [Namespaces](https://secure.php.net/manual/en/language.namespaces.php). 
+  
+## Pressbooks Coding Recommendations (Optional)
 
-1. `magic`
-2. `public`
-3. `protected`
-4. `private`
-5. `static`
+Write accurate [PHPDoc](http://en.wikipedia.org/wiki/PHPDoc) styled code comments.
 
 Prefix WP Post meta keys with `pb_`.
+
 Prefix WP User meta keys with `pb_`.
-Prefix WP Option names with `pressbooks_`.
 
```

Comment:
Can you add a line about prefixing [action and filter hook](https://developer.wordpress.org/plugins/hooks/) names with `pb_`?